internal/interactor/deployment.go

```diff
@@ -92,16 +98,21 @@ func (i *Interactor) Deploy(ctx context.Context, u *ent.User, r *ent.Repo, d *en
 }
 
 func (i *Interactor) IsApproved(ctx context.Context, d *ent.Deployment) bool {
-	as, _ := i.ListApprovals(ctx, d)
+	rvs, _ := i.ListReviews(ctx, d)
+
+	for _, r := range rvs {
+		if r.Status == review.StatusRejected {
+			return false
+		}
+	}
 
-	approved := 0
-	for _, a := range as {
-		if a.Status == approval.StatusApproved {
-			approved = approved + 1
+	for _, r := range rvs {
+		if r.Status == review.StatusApproved {
+			return true
 		}
 	}
 
-	return approved >= d.RequiredApprovalCount
+	return false
```

Comment:
If there is a rejection, it blocks to deploy to GitHub. It needs at least one approval to deploy.